src/test/java/seedu/anilist/logic/commands/CommandResultTest.java

```diff
@@ -13,23 +11,26 @@ public void equals() {
         CommandResult commandResult = new CommandResult("feedback");
 
         // same values -> returns true
-        assertTrue(commandResult.equals(new CommandResult("feedback")));
-        assertTrue(commandResult.equals(new CommandResult("feedback", false, false)));
+        assertEquals(commandResult, new CommandResult("feedback"));
+        assertEquals(commandResult, new CommandResult("feedback", false, false));
 
         // same object -> returns true
-        assertTrue(commandResult.equals(commandResult));
+        assertEquals(commandResult, commandResult);
 
         // null -> returns false
-        assertFalse(commandResult.equals(null));
+        assertNotEquals(commandResult, null);
 
         // different types -> returns false
-        assertFalse(commandResult.equals(0.5f));
+        assertNotEquals(commandResult, 0.5f);
 
         // different feedbackToUser value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("different")));
+        assertNotEquals(commandResult, new CommandResult("different"));
 
         // different exit value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("feedback", false, true)));
+        assertNotEquals(commandResult, new CommandResult("feedback", false, true));
+
+        // different stats value -> returns false
+        assertNotEquals(commandResult, new CommandResult("feedback", true, false));
```

Comment:
A bit of magic strings used here, perhaps can put "feedback" as a constant?

Reply:
ok

Reply:
added constants, but cant name them all caps.